File: credit/credit.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    long number = get_long("Number: ");
    int num;
    int sum = 0;
    while(number > 0)
    {
        num = number % 10;
        number = (number - num)/10;
        sum += num*2;
    }
    printf("%i\n", sum);

}

File: sentimental-credit/credit.py


number = input('Number: ')
numbers = list()
for i in number:
    numbers.append(int(i))
sum = 0
for j in range(-2,-len(numbers) - 1,-2):
    num = numbers[j] * 2
    sum += (num / 10) + (num % 10) if num > 9 else
print(sum)





